### DISCUSSION TITLE
How change height when autoresize plugin enabled

### DISCUSSION THREAD

**Discussion by MikePrus** [tinymce/tinymce#9113] [General]
I use TinyMCE editor with autoresize plugin, so my component looks like this:
```jsx
<Editor
        onEditorChange={onEditorChange}
        value={value}
        onKeyDown={onKeyDown}
        onFocus={onFocus}
        init={{
          type: 'textarea',
          apiKey: import.meta.env.VITE_TINYMCE_API_KEY,
          placeholder: placeholder,
          plugins: 'autoresize',
          max_height: 150,
          min_height: 50,
          toolbar: false,
          menubar: false,
          branding: false,
          statusbar: false,
          resize: false,
          content_style: 'body { font-family: Greycliff CF,Arial,sans-serif; color: #424647; font-size:14px }',
        }}
      />
```
I want to change the default height to make it shorter according to design, and faced a problem because `min_height` and `max_height` props don't work.

Maybe somebody can help me!

**Comment by lostkeys** [comment]
Hi @MikePrus 

Add `autoresize_bottom_margin: 0` to your config and it will work. [TinyMCE adds a 50px bottom margin](https://www.tiny.cloud/docs/tinymce/6/autoresize/#autoresize_bottom_margin) to the content by default.

**Reply by MikePrus** [reply]
@lostkeys man you saved my day 🙌. I've spent half of the day reading docs, I almost gave up. Thanks a lot!
